apache/superset
# test: Cypress 7.x migration

### SUMMARY
This PR migrates Cypress to 7.x. Details for this process can be found [here](https://docs.cypress.io/guides/references/migration-guide#Migrating-to-Cypress-7-0).

### TESTING INSTRUCTIONS
- Navigate to `superset-frontend/package.json` and observe the dev dependency: `"@cypress/react": "^5.10.0"`
- Navigate to `superset-frontend/cypress-base/package.json` and observe the dependency: `"@cypress/code-coverage": "^3.9.11"` and dev dependency: `"cypress": "^7.0.0"`
- Run the Cypress tests and observe that tests have been properly migrated.

### ADDITIONAL INFORMATION
